python/metatensor-learn/metatensor/learn/nn/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .module_map import ModuleMap, Linear  # noqa
```

Comment:
Generally, I would recommend to store different module implementations (as `Linear`) in separate files

Reply:
separated the files